### ISSUE TITLE
[2026/04/29] Large Language Diffusion Models

### ISSUE THREAD

**Issue by zaemyung** [zaemyung/Seeking-SOTA#224]
- Large Language Diffusion Models
- [Shen Nie](https://arxiv.org/search/cs?searchtype=author&query=Nie,+S), [Fengqi Zhu](https://arxiv.org/search/cs?-searchtype=author&query=Zhu,+F), [Zebin You](https://arxiv.org/search/cs?searchtype=author&query=You,+Z), [Xiaolu Zhang](https://arxiv.org/search/cs?searchtype=author&query=Zhang,+X), [Jingyang Ou](https://arxiv.org/search/cs?searchtype=author&query=Ou,+J), [Jun Hu](https://arxiv.org/search/cs?searchtype=author&query=Hu,+J), [Jun Zhou](https://arxiv.org/search/cs?searchtype=author&query=Zhou,+J), [Yankai Lin](https://arxiv.org/search/cs?searchtype=author&query=Lin,+Y), [Ji-Rong Wen](https://arxiv.org/search/cs?searchtype=author&query=Wen,+J), [Chongxuan Li](https://arxiv.org/search/cs?searchtype=author&query=Li,+C)
- https://arxiv.org/abs/2502.09992
> The capabilities of large language models (LLMs) are widely regarded as relying on autoregressive models (ARMs). We challenge this notion by introducing LLaDA, a diffusion model trained from scratch under the pre-training and supervised fine-tuning (SFT) paradigm. LLaDA employs a forward data masking process and a reverse generation process, parameterized by a Transformer to predict masked tokens. It provides a principled generative approach for probabilistic inference by optimizing a likelihood lower bound. Across extensive benchmarks on general tasks, math, code, and so on, LLaDA demonstrates strong scalability and performs comparably to our self-constructed ARM baselines. Remarkably, LLaDA 8B is competitive with strong LLMs like LLaMA3 8B in in-context learning and, after SFT, exhibits impressive instruction-following abilities in case studies such as multi-turn dialogue. Moreover, LLaDA addresses the reversal curse, surpassing GPT-4o in a reversal poem completion task. Our findings show the promise of diffusion models for language modeling at scale and challenge the common assumption that core LLM capabilities discussed above inherently depend on ARMs. Project page and codes: [this https URL](https://ml-gsai.github.io/LLaDA-demo/).
